recipe/meta.yaml

```diff
@@ -1,34 +1,47 @@
 {% set name = "slycot" %}
 {% set version = "0.3.3" %}
-{% set sha256 = "d98ec859a398af8a3917a0d62e0724996c8b3e780813f8685410525aed9fbafd" %}
+{% set sha256 = "e93f15b0e1c95262e3b498b0f378eb5302f7f02ccb720362840cbd0e13780565" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # temporary, for testing. 
+  url: https://github.com/repagh/Slycot/archive/update-buildconfig.tar.gz
+  # url: https://github.com/python-control/Slycot/archive/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 1003
-  # NOTE: Conda forge lacks openblas on Windows, so skip the Windows build.
-  skip: true  # [win]
+  number: 1004
+  # problem with python2.7 configuration on windows and osx
+  skip: true                     # [win and py==27]
+  skip: true                     # [osx and py==27]
+
+# TODO: (later) solve imperfections
+# * the clang compiler on osx gives a link error, thought to be related to 
+#   cmake and env. Try later when this may be fixed. Alternative: gcc
+# * the mkl library does not link on osx or win. Alternative: openblas
 
 requirements:
   build:
-    - {{ compiler('fortran') }}
-    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}  # [not win]
+    - flang                      # [win]
+    - {{ compiler('c') }}        # [not osx]
+    - gcc                        # [osx]
 
   host:
-    - libblas
-    - liblapack
+    - mkl                        # [linux]
```

Comment:
I would suggest that mkl somehow be an option for the user. I personally wouldn't want to require this.

Reply:
Hm. 

How can a user select that when installing from conda-forge?
And then we need to add/build more variants.

With the osx build I hit a wall trying to use clang; I reverted to gcc instead, that at least works. I found some vague bug reports on missing development libraries, which is what gets reported when trying clang. The osx script seems not to work with mkl.

The windows and Linux scripts install mkl, also when trying to build other (openblas) variants. I guess these are pulled in by numpy then?

Reply:
They way it works on linux systems is this: https://stackoverflow.com/questions/33873211/is-it-possible-to-switch-between-blas-libraries-without-recompiling-program

I'm not sure how conda forge is handling this now. Everything used to be all based on openblas.

Reply:
Looks like you are right. I see that the mkl stuff is from anaconda, and conda-forge seems to stick to openblas. 
I converted the meta.yaml to use openblas for all platforms. 

I Also squashed the build modifications in the Slycot PR. Can you merge that and tag 0.3.4? After that I can update the url for this PR, squash, and we can finally have a windows conda-forge package too.

Reply:
> I Also squashed the build modifications in the Slycot PR. Can you merge that and tag 0.3.4? After that I can update the slycot source url in the meta.yaml, squash, and we can finally have a windows conda-forge package too.

I don't think I have merge permissions on Slycot.

Reply:
I don't.